ckanext/vocabulary_services/helpers.py

```diff
@@ -37,3 +39,11 @@ def scheming_vocabulary_service_choices(field):
             log.error(str(e))
 
     return choices
+
+
+def check_access(context):
+    # `config_option_update` only accessible to sysadmin users
+    try:
+        logic_check_access('config_option_update', context, {})
+    except Exception as e:
+        abort(404, 'Not found')
```

Comment:
Should this be a 403 forbidden status code from the NotAuthorized exception, or is there reason it needs to be 404?

Reply:
yeah I made it 404 so that it appears to not exist for anyone trying to access it that doesn't have permission in case they thought it was a good idea to keep trying it for some reason